recipes/Time_Series/Preprocessor_Use_and_Performance_Evaluation.ipynb

```diff
@@ -870,12 +870,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 78,
+   "execution_count": 14,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/wmgifford/miniconda3/envs/granite-tsfm/lib/python3.12/site-packages/pandas/core/indexes/base.py:7588: FutureWarning: Dtype inference on a pandas object (Series, Index, ExtensionArray) is deprecated. The Index constructor will keep the original dtype in the future. Call `infer_objects` on the result to get the old behavior.\n",
+      "  return Index(sequences[0], name=names)\n"
+     ]
+    },
```

Comment:
@wgifford let's keep these warnings out if possible.

Reply:
Yes, will look into it.